lib/features/stories/view/screens/crop_image_screen.dart

```diff
@@ -116,30 +115,37 @@ class _CropImageScreenState extends State<CropImageScreen> {
             mainAxisAlignment: MainAxisAlignment.spaceAround,
             children: [
               IconButton(
+                  key: const ValueKey('refresh_key'),
```

Comment:
We don't directly put the keys in this way.
we instantiate the keys in the lib/core/constants/keys.dart file and then use it in the needed location.
otherwise, Eslam might scream, and this is a serious problem; you don't want a headache, do you